pyproject.toml

```diff
@@ -32,9 +32,6 @@ classifiers = [
         'Topic :: Scientific/Engineering :: Chemistry',
 ]
 
-[project.scripts]
```

Comment:
This should not be deleted? Please check, and check if `package update` can be made clever enough to handle this.